### PR TITLE
Common integration for fty-test

### PR DESCRIPTION
**Pull request by bstepankova**

### CODE REVIEW

File: src/web/src/assets_in.ecpp [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<#
+p<#
```
Comment (EldoreiJK) [inline src/web/src/assets_in.ecpp]: I think this might be a typo

File: src/web/src/location_helpers.cc [hunk 2]
```diff
@@ -26,10 +26,11 @@
  * \brief Not yet documented file
  */
 #include "web/src/location_helpers.h"
-
+#include <fty_common_db_dbpath.h>
```
Comment (EldoreiJK) [inline src/web/src/location_helpers.cc]: Why is this header needed?

File: .travis.yml [hunk 3]
```diff
@@ -1,7 +1,6 @@
 # Travis CI script
 # This is a skeleton created by zproject.
 # You can add hand-written code here.
-### Note: Customized after generation; take care upon regens
```
Comment (jimklimov) [inline .travis.yml]: Why is this and lower comments about customizations removed? Customizations themselves are still needed and still here...

File: .travis.yml [hunk 4]
```diff
@@ -162,10 +156,9 @@ before_install:
 
 ### Note: Fix up SASL pkgconfig before build
 before_script:
-- ./.travis.fixsasl
```
Comment (jimklimov) [inline .travis.yml]: I think the leading `-` was needed here, as it is a YAML list (of operations to run)

File: ci_build.sh [hunk 5]
```diff
@@ -4,17 +4,9 @@
 #  THIS FILE IS 100% GENERATED BY ZPROJECT; DO NOT EDIT EXCEPT EXPERIMENTALLY  #
 #  READ THE ZPROJECT/README.MD FOR INFORMATION ABOUT MAKING PERMANENT CHANGES. #
 ################################################################################
-# NOTE: This script was customized (PATH, added commented-away quicktest lines,#
```
Comment (jimklimov) [inline ci_build.sh]: Customizations below are lost. Some of these might be responsible for travis fails of this current mix of legacy+generated codebase.

File: packaging/debian/control [hunk 6]
```diff
@@ -1,6 +1,5 @@
 #
 #    fty-rest - Common core REST API for 42ity project
-# Note: this file is customized after zproject generation, be sure to keep it
```
Comment (jimklimov) [inline packaging/debian/control]: Customization lost

File: packaging/debian/libfty-rest-dev.install [hunk 7]
```diff
@@ -1,8 +1,3 @@
-# Note: this file is customized after zproject generation, be sure to keep it
```
Comment (jimklimov) [inline packaging/debian/libfty-rest-dev.install]: Customization lost

File: packaging/debian/libfty-rest1.install [hunk 8]
```diff
@@ -1,5 +1 @@
-# Note: this file is customized after zproject generation, be sure to keep it
```
Comment (jimklimov) [inline packaging/debian/libfty-rest1.install]: Customization lost

File: packaging/redhat/fty-rest.spec [hunk 9]
```diff
@@ -1,6 +1,5 @@
 #
 #    fty-rest - Common core REST API for 42ity project
-#    NOTE: This file was customized after generation, be sure to keep it
```
Comment (jimklimov) [inline packaging/redhat/fty-rest.spec]: Customization lost

File: src/Makemodule-local.am [hunk 10]
```diff
@@ -1,4 +1,4 @@
-# Custom targets, not managed via zproject
+# Custom targets, not managed via  zproject
```
Comment (jimklimov) [inline src/Makemodule-local.am]: Extra space :)

**Pull request merged**